### ISSUE TITLE
XGBRegressor Importance_type

### ISSUE THREAD

**Issue by GustavoGL** [dmlc/xgboost#4094]
Thanks for participating in the XGBoost community! We use https://discuss.xgboost.ai for any general usage questions and discussions. The issue tracker is used for actionable items such as feature proposals discussion, roadmaps, and bug tracking.  You are always welcomed to post on the forum first :)

Issues that are inactive for a period of time may get closed. We adopt this policy so that we won't lose track of actionable issues that may fall at the bottom of the pile. Feel free to reopen a new one if you feel there is an additional problem that needs attention when an old one gets closed.

For bug reports, to help the developer act on the issues, please include a description of your environment, preferably a minimum script to reproduce the problem.

For feature proposals, list clear, small actionable items so we can track the progress of the change.

**Comment by GustavoGL** [comment]
I am creating a model with XGBRegressor and I noticed that the attribute *importance_type*, which is set to "gain" by default always uses type "weight", no matter the specified value.
This attribute sets the feature importance type for the *feature_importances_* property.

To reproduce the issue, try creating two XGBRegressor models with the same parameters using different values for the *importance_type* attribute. Then, call the property *feature_importances_* and the result obtained is the same. 

I have checked in the definition of this property and it seems to be using the *get_fscore()* method of the XGBModel.getbooster() tree model. This function always uses *importance_type*="weight", hard-coded.

I am using XGBoost version 0.81 installed with conda.
Python: 3.6
Operating system: MacOS Mojave version 10.14.2

**Comment by hcho3** [comment]
The fix was made in #3876, and it is not part of version 0.81. Can you compile from the source?

```
git clone --recursive https://github.com/dmlc/xgboost
cd xgboost
mkdir build
cd build
CC=gcc-8 CXX=g++-8 cmake ..
make -j4
cd ../python-package
python setup.py install
```

**Comment by GustavoGL** [comment]
Ok. Thanks!
I am closing the issue then.
